stable/aws-for-fluent-bit/values.yaml

```diff
@@ -323,6 +328,27 @@ volumeMounts:
     mountPath: /var/lib/docker/containers
     readOnly: true
 
+# To turn on liveness and readiness probes .Values.ServiceMonitor is requred to be enabled
+# and .Values.serviceMonitor.service must also be populated with type,port,targetPort 
+# and to enable the healthcheck endpoint set .Values.service.extraService
+livenessProbe:
```

Comment:
Do we want to enable this by default? I think it could be an example that users enable if they want, but not turned on by default. 

If FLB is unable to send, this probe will fail and it will be restarted. 

Please add a note that un-commenting the lines for the Health_Check in the Service section is necessary.

Reply:
I think enabling this and causing it to restart is an appropriate action to ensure that the service is always running correctly.  The issue we ran into was a completely silent issue which we didnt find for almost a week - the pod failed to send logs but just sat in a limbo state doing nothing.

Reply:
Will get a note added to the Service Section stating its required for liveness/readiness probe

Reply:
I see, interesting to learn that upstream enables it... I am fine with enabling by default here too. However, if we choose that we also need to enable `Health_Check On` by default otherwise it won't work... so you need to uncomment that in the values.yaml right?

Reply:
It only enabled by default if you also turn on serviceMonitor - if you would like me to enable that by default and have the liveness probe on by default I’m happy to make that change

Reply:
I see... I think I need to understand this more and either way we need very very clear docs in the readme. Preferably with an example. What is serviceMonitor? I don't see that in the current readme: https://github.com/aws/eks-charts/tree/master/stable/aws-for-fluent-bit